scylla-macros/src/serialize/row.rs

```diff
@@ -341,10 +341,10 @@ impl Generator for ColumnSortingGenerator<'_> {
         };
 
         parse_quote! {
-            fn serialize<'b>(
+            fn serialize<'_scylla_ser_row_writer_buffer>(
                 &self,
                 ctx: &#crate_path::RowSerializationContext,
-                writer: &mut #crate_path::RowWriter<'b>,
+                writer: &mut #crate_path::RowWriter<'_scylla_ser_row_writer_buffer>,
```

Comment:
:100: It's shocking that we wrote it this way. Thanks for spotting the problem!

Reply:
Probably doable but I am unsure if the effort is worth it. From my understanding it is common practice in macros to use underscores when wanting to specify a lifetime that is more guaranteed to not clash.